Jenkinsfile.release

```diff
@@ -523,6 +555,51 @@ echo ===============================================
     timeout: [time: 10, unit: 'MINUTES']
   )
 
+  pipeline.createStage(
+    name             : "Publish Docker Image",
+    isSkippable      : true,
+    displayTimestamp : true,
+    timeout          : [time: 120, unit: 'MINUTES'],
+    shouldExecute    : {
+      return isFormalRelease && (releaseArtifacts['docker-amd64'] || releaseArtifacts['docker-s390x'])
+    },
+    stage            : {
+      withCredentials([usernamePassword(
+        credentialsId: 'ZoweDockerhub',
+        usernameVariable: 'USERNAME',
+        passwordVariable: 'PASSWORD'
+      )]) {
+        def publishCommands = "docker login -u \"${USERNAME}\" -p \"${PASSWORD}\"\n"
+        def manifestAmends = ""
+        // load images
+        if (releaseArtifacts['docker-amd64']) {
+          sh "docker load --input .release/zowe-v1-lts.amd64-*.tar"
+          if ("${USERNAME}" != "ompzowe") {
+            // the image in tar should always be ompzowe/zowe-v1-lts:amd64
+            sh "docker tag ompzowe/zowe-v1-lts:amd64 ${USERNAME}/zowe-v1-lts:amd64"
+          }
+          publishCommands += "docker push ${USERNAME}/zowe-v1-lts:amd64\n"
+          manifestAmends += " --amend ${USERNAME}/zowe-v1-lts:amd64"
+        }
+        if (releaseArtifacts['docker-s390x']) {
+          sh "docker load --input .release/zowe-v1-lts.s390x-*.tar"
+          if ("${USERNAME}" != "ompzowe") {
+            // the image in tar should always be ompzowe/zowe-v1-lts:s390x
+            sh "docker tag ompzowe/zowe-v1-lts:s390x ${USERNAME}/zowe-v1-lts:s390x"
+          }
+          publishCommands += "docker push ${USERNAME}/zowe-v1-lts:s390x\n"
+          manifestAmends += " --amend ${USERNAME}/zowe-v1-lts:s390x"
+        }
+
+        // publish images
+        sh publishCommands +
+          "export DOCKER_CLI_EXPERIMENTAL=enabled\n" +
+          "docker manifest create ${USERNAME}/zowe-v1-lts:latest ${manifestAmends}\n" +
```

Comment:
in addition to `latest`, this is a great opportunity to also create version tags from `${params.ZOWE_RELEASE_VERSION}`
that way, latest and v1.18.0 both point to the particular s390x and amd64 releases.